### ISSUE TITLE
Lab 1 Report Feedback

### ISSUE THREAD

**Issue by sywong888** [ezh247467/cse15l-lab-reports#1]
Here is your feedback for Lab Report 1:

- [x] All relevant URLs written as markdown links
- [x] All relevant code examples written with backticks (```)
- [x] All screenshots correspond to the described steps
- [x] All titles of sections are formatted as headers or similar
- [] Lab report is on its own page, not the index page
The lab report is on the index page when it should be on a separate page called something like lab-report-1. You should add a link on your index page that links to your lab-report-1 page.
- [x] Last example shows how to run remotely saving time
- [x] Text is formatted in a way that was easy for grader to read

Additional comments from your grader:
Great job! Just make sure to fix the issue regarding putting the lab report on its own page.

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by ezh247467** [comment]
Hi, I am confused about how to put the lab report on its own page. I have my lab report as a separate file as lab-report-1.md, but I am unsure how to proceed from there.

**Comment by sywong888** [comment]
On your index page, you need to add a link that directs to your lab report page (lab-report-1.md). The "Setting Up a Web Page" section of the [Lab 2 Writeup](https://docs.google.com/document/d/1Nw6gdehL-BzqjeVV1jzi_Ni4cdpx2uquLztLGTdzUdU/edit#heading=h.w9k652xekw0x) has some helpful hints on how to do this. If you still have questions on how to proceed feel free to ask me or ask tutors during their tutor hours!

**Comment by ezh247467** [comment]
I see. I tried doing that now, is what I did sufficient to get full credit or am I missing anything else?

**Comment by ezh247467** [comment]
Also, it seems that since my lab report 1 has not been "graded" on Gradescope, I cannot submit a regrade request. How should I proceed from here?

**Comment by sywong888** [comment]
Yes the link you added is sufficient. You can submit a regrade request starting tomorrow. There are more details in my most recent post on Piazza.

**Comment by ezh247467** [comment]
Hi, I have checked my GradeScope and I still have not gotten my grade for the lab report. Is there something I am missing?

**Comment by sywong888** [comment]
We have not opened regrade requests, but we will soon. I apologize for the delay.

**Comment by ezh247467** [comment]
Hi, I just wanted to voice a concern. According to Gradescope, the lab report 1 assignment will be closing and I have not gotten my regrade request answered yet. Will my lab report still be able to be regraded after the assignment closes on GradeScope?

**Comment by sywong888** [comment]
Graders will only respond to regrade requests after the regrade period closes (EOD Monday). So until the end of today you can submit a regrade request - you just won't receive a response until tomorrow or later.

**Comment by ezh247467** [comment]
I see. Thank you so much for the clarification!

**Comment by sywong888** [comment]
Great job making the separate page!
